### PR TITLE
PLG-676-PLG-794-investigate-which-steps-to-include-in-react-native-sample-app-workflows

### PR DESCRIPTION
**Pull request by hisaac**
The changes here are an attempt to make the app's workflow more "React Nativey", by using `npm` steps over `yarn`, and utilizing some of our React Native-specific steps as well.

## Changes

- Replace `yarn` step with `npm` step
- Add React Native Bundle step to demonstrate bundling
- Reformat `bitrise.yml` to have less duplicate code, and to use more React Native-specific steps
- Rename a couple files/directories that got missed in https://github.com/bitrise-io/getting-started-react-native-sample-app/pull/3

### CODE REVIEW

File: android/gradle.deps [hunk 1]
```diff
@@ -0,0 +1 @@
+b8369299bb6500f98fce82fbc727a900cd5e4bdecc6cd0103debc39ce2c41455b9fa3e0d77999584aaa9aee9eb6703fb
```
Comment (jakerobb) [inline android/gradle.deps:1]: You should configure your IDE to always ensure the presence of a newline at the end of a file.

File: bitrise.yml [hunk 2]
```diff
@@ -3,88 +3,69 @@ format_version: "11"
 default_step_lib_source: https://github.com/bitrise-io/bitrise-steplib.git
 project_type: react-native
 
+trigger_map:
+- push_branch: "*"
+  workflow: primary
```
Comment (jakerobb) [inline bitrise.yml:8]: Edited the PR title to reference PLG-794. I'm not sure, but I'm hoping that will trigger the automatic integration.

**Review by jakerobb — APPROVED** [review]
LGTM!

File: android/gradle.deps (hunk 1 above)
Comment (hisaac) [reply android/gradle.deps:1]: Yep! Already is. This file was auto-generated by gradle, so I didn't edit it manually.

File: bitrise.yml (hunk 2 above)
Comment (hisaac) [reply bitrise.yml:8]: ah I was wondering how the title got changed 😅 Sounds good.

Comment (jakerobb) [reply bitrise.yml:8]: I'm pleased to report that it _did_ trigger the integration! 🙂

**Pull request merged**